src/Stripe.net/Entities/Invoices/Invoice.cs

```diff
@@ -541,6 +541,9 @@ public PaymentIntent PaymentIntent
         internal ExpandableField<PaymentIntent> InternalPaymentIntent { get; set; }
         #endregion
 
+        [JsonProperty("payment_settings")]
```

Comment:
No docstring is that expected?

Reply:
No. I think this problem might extend to more expandable properties, too. Will open an internal ticket to investigate.